ankane/ahoy.js
# Fix caching issues when trying to get DOM element's data-section attribute value

### What I did
Instead of using jQuery's `.data` function, use the `.attr` function to prevent caching issue with elements that their data-section is changed dynamically.
Also bumped the version number.

#### How to reproduce the issue this solves:
1. Set data-section on any DOM element to some value. 
2. Click on it once.
3. Change it's data-section value to something else. 
4. Click on it again.
5. You will see that the second data-section you are getting in the event is the **old** data-section.

I already use this and it works perfectly.
